packages/controlled-form/src/lib/Input.tsx

```diff
@@ -29,7 +43,7 @@ export const ControlledInput = ({
       defaultValue={defaultValue}
       disabled={disabled}
       rules={{
-        required,
+        required: typeof required === 'boolean' ? undefined : required,
```

Comment:
Not sure I understand this. If `required` is `true` you just ignore it for the validation?